lib/fluent/plugin/out_webhdfs.rb

```diff
@@ -66,6 +66,10 @@ class Fluent::Plugin::WebHDFSOutput < Fluent::Plugin::Output
   config_param :kerberos, :bool, default: false
   desc 'kerberos keytab file'
   config_param :kerberos_keytab, :string, default: nil
+  desc 'Use delegation token while upload webhdfs or not'
+  config_param :reuse_delegation_token, :bool, default: false
+  desc 'delegation token reuse timer in hour (default 8)'
+  config_param :renew_kerberos_delegation_token_time_hour, :integer, default: 8
```

Comment:
How about using `:time` instead of `integer` and remove `_hour` suffix?
By this type, you can use values like `8h` (8 hour), `1d` (24hour) and so on.
(In this case you need to convert it to hours before pass it to `WebHDFS::Client.new` though).